certbot/client.py

```diff
@@ -150,8 +150,14 @@ def perform_registration(acme, config):
         return acme.register(messages.NewRegistration.from_data(email=config.email))
     except messages.Error as e:
         if e.typ == "urn:acme:error:invalidEmail":
-            config.namespace.email = display_ops.get_email(invalid=True)
-            return perform_registration(acme, config)
+            if config.noninteractive_mode:
+                msg = ("The email you specified was unable to be verified "
```

Comment:
"Unable to be verified" seems a bit confusing... Maybe a first sentence like 'The ACME server believes {email} is an invalid email address'.